src/shared/util.js

```diff
@@ -1482,6 +1498,7 @@ MessageHandler.prototype = {
     let streamId = data.streamId;
 
     let sendStreamResponse = ({ stream, success, reason, }) => {
+      reason = reason === undefined ? reason : wrapReason(reason);
```

Comment:
since we are adding reason check at wrapReason, let's remove it from here and add to callsites instead